scripts/test/SceneBuilder.ps1

```diff
@@ -0,0 +1,300 @@
+   $UnityPath = ${Env:${{ parameters.UnityVersion }}}
+   # Find unity.exe as Start-UnityEditor currently doesn't support arbitrary parameters
+   $editor = Get-ChildItem $UnityPath -Filter 'Unity.exe' -Recurse | Select-Object -First 1 -ExpandProperty FullName
+   
+   # The build output goes to a unique combination of Platform + Arch + Scene to ensure that
+   # each build will have a fresh destination folder.
+   $outDir = "$(Build.ArtifactStagingDirectory)\build\${{ parameters.UnityVersion }}\${{ parameters.Platform }}_${{ parameters.Arch }}_${{ parameters.Scene }}"
+   $logFile = New-Item -Path "$outDir\build\build.log" -ItemType File -Force
+   $logDirectory = "$outDir\logs"
+   $projectPath = $(Get-Location)
+   $sceneList = ''
+   If ("${{ parameters.Scene }}" -eq "HandInteractionExamples")
```

Comment:
nit: convention is to use "if" vs "If"

Reply:
I changed all If, Else, and ElseIf to if, else, and elseif in the pipelines and script folders.